src/ontology/ro-edit.owl

```diff
@@ -5127,6 +5148,7 @@ SubObjectPropertyOf(obo:RO_0002639 obo:RO_0002635)
 # Object Property: obo:RO_0002640 (intracellular endoparasite of)
 
 AnnotationAssertion(obo:IAO_0000115 obo:RO_0002640 "A sub-relation of endoparasite-of in which the parasite inhabits host cells.")
+AnnotationAssertion(obo:IAO_0000119 obo:RO_0002640 <https://en.wikipedia.org/wiki/Communal_roosting>)
```

Comment:
Note that this annotation is not part of @JCGiron request, but an artefact from me reconciling the documentations. Please ignore during review - we want this annotation back independent of this pull request.

Reply:
Oh it was wrong, I removed it.